docs/static/include/pluginbody.asciidoc

```diff
@@ -910,12 +910,12 @@ Logstash plugins:
 [subs="attributes"]
 ----------------------------------
   # Gem dependencies
-  s.add_runtime_dependency 'logstash-core', '>= 1.4.0', '< 2.0.0'
+  s.add_runtime_dependency 'logstash-core', '>= 5.0.0', '< 6.0.0'
   s.add_development_dependency 'logstash-devutils'
 ----------------------------------
 This gemspec has a runtime dependency on the core Logstash gem and requires that
-it have a version number greater than or equal to version 1.4.0 `'>= 1.4.0'`,
-and less than version 2.0 `'< 2.0.0'`.
+it have a version number greater than or equal to version 5.0.0 `'>= 5.0.0'`,
```

Comment:
This should be dependency on logstash-core-plugin-api with constraints: ">= 1.60", "<= 2.99"